internal/types/types.go

```diff
@@ -50,12 +51,23 @@ func (u UniquePath) RelativePath() (string, error) {
 	return rPath, nil
 }
 
-// DisplayPath represents Slash-separated path to the package directory on the filesytem relative
-// to current working directory.
+// DisplayPath represents os-agnostic Slash-separated path to the package directory on the filesystem relative
+// to parent directory of root package on which the command is invoked.
+// root package is defined as the package on which the command is invoked by user
 // This is not guaranteed to be unique (e.g. in presence of symlinks) and should only
 // be used for display purposes and is subject to change.
 type DisplayPath string
 
+// NewDisplayPath returns the os-agnostic Slash-separated path for display purposes
+// the input path must be a relative path, returns error if input path is absolute
```

Comment:
The semantic of input is still unclear.

Relative to what? Is it OS path or slash-seperated? Provide examples.

Reply:
Removed this method as a whole.